cetmix_tower_server/models/cx_tower_plan_log.py

```diff
@@ -48,9 +48,7 @@ class CxTowerPlanLog(models.Model):
         comodel_name="cx.tower.command.log", inverse_name="plan_log_id", auto_join=True
     )
     plan_status = fields.Integer(string="Status")
-    parent_flight_plan_log_id = fields.Many2one(
-        "cx.tower.plan.log",
-    )
+    parent_flight_plan_log_id = fields.Many2one("cx.tower.plan.log", string="Main Log")
```

Comment:
I agree regarding the `ondelete="cascade"` however constraint for the logs is not needed because parent/children relations are regulated on the flight plan level

Reply:
@dessanhemrayev please add `ondelete="cascade"` here